src/Category/PERM.agda

```diff
@@ -0,0 +1,65 @@
+{-# OPTIONS --type-in-type #-}
+
+module Category.PERM where
+
+open import Categories
+open Category
+open import Relation.Binary.PropositionalEquality as Eq using (_≡_; refl; cong; sym; trans)
+open Eq.≡-Reasoning using (begin_; _≡⟨⟩_; _∎; step-≡)
+open import Relation.Binary.Structures
+
+import Isomorphisms
+open Isomorphisms.Isomorphism
+open import Category.SET
+
+open import Isomorphisms SET
+
+
+record PermObj : Set where
+  field
+    permCarrier : Set
```

Comment:
As a stylistic note: we have records that work in Agda, so prefixing the fields of a record with the name of the record isn't required.